packages/state-manager/src/sagas/communities/saveCommunityMetadata/saveCommunityMetadata.saga.ts

```diff
@@ -22,4 +23,12 @@ export function* saveCommunityMetadataSaga(
       ownerCertificate: action.payload.ownerCertificate,
     })
   )
+
+  const community = yield* select(communitiesSelectors.currentCommunity)
+  if (!community) return
+  const isOwner = community.CA
+
+  if (!isOwner) {
+    yield* put(publicChannelsActions.sendUnregisteredInfoMessage())
+  }
```

Comment:
I guess you can simplify it as just
```
const community = yield* select(communitiesSelectors.currentCommunity)
if (community?.CA) return
yield* put(publicChannelsActions.sendUnregisteredInfoMessage())
```

Reply:
Although the cleanest solution would be to make a dedicated saga, hooking up on `communitiesActions.saveCommunityMetadata` or some other action (eg. replicating channels as it's required to send a message)

Reply:
I really like your solution, it's very clean, only 3 lines of code